ts.js

```diff
@@ -113,6 +113,7 @@
       ],
       '@typescript-eslint/type-annotation-spacing': 'error',
       '@typescript-eslint/unified-signatures': 'error',
+      'consistent-return': 'off',
```

Comment:
so your comment in desc:

> Disable consistent-return rule in TS....

implies that disabling this will still put us in a position where we will get nagged when we have inconsistent return types in functions (like where we bail early with a bare 'return' but the success case actually returns an expected type?).

because, tbh, i like this nag, and prefer disabling it in the rare case where we have some short-circuit return that has a justified case to not return data with it.

or are you saying, that the required defined return type will require you to be aware when you are returning a possible 'null' value because the detection will infer that your return should be `null|TheObjectType`.

Reply:
The latter. `@typescript-eslint/explicit-function-return-type` requires that you have a defined return type and follow it, so if you're not following it, you'll get yelled at. So it does everything that `consistent-return` does and more